tools/m2-repo-cleaner/src/main/java/org/fusesource/ide/tools/mavenutils/M2RepoCleaner.java

```diff
@@ -101,4 +118,12 @@ private static void handleCorruptedZip(File file) {
 			}
 		}
 	}
+	
+	private static void handleLastUpdatedFile(File file) {
+		handleCorruptedZip(file);
+//		corruptedFiles.add(file.getPath());
```

Comment:
470 is quite huge, do we know which files they are?
would it be possible that it is all artefacts that are available on staging repositories?